### PR TITLE
binary needs to be called main.bin

### PR DESCRIPTION
**Pull request by stappersg**
Very raw note,  preventing that the hard way earned knowledge gets lost.

### CODE REVIEW

**Comment by a2800276** [comment]
Did you initially install an app with the same name from the hatchery? I can't reproduce this here, could you please try installing with a different name, i.e.:

instead of
```
$ webusb_push.py --run rust rust_esp.img
```
e.g. this:
```
$ webusb_push.py --run rust9876986 rust_esp.img
```

Gut feeling would be that maybe the appfs checks that the filename is the same as for the previous version.


I'm afraid maybe the note is a bit too raw for docs and may not make sense to anyone not involved in the issue, we should probably rewrite as prose instead of a raw irc log for it to be more useful.

**Comment by stappersg** [comment]
On Mon, Aug 01, 2022 at 01:56:36AM -0700, Tim Becker wrote:
> 
> I'm afraid maybe the note is a bit too raw for docs and may not make
> sense to anyone not involved in the issue, we should probably rewrite
> as prose instead of a raw irc log for it to be more useful.

True.  This merge request is now "input" for the rewrite.

 
Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by a2800276** [comment]
#95
